File: Codigo/CJF.Sgcpj.App/CJF.Sgcpj.App/src/modules/oficialia/data/insertar-promocion.js
import { ref } from "vue";
export class InsertarPromocion {
  asuntoNeunId = ref(0);
  tipoCuaderno = ref(0);
  fechaPresentacion = ref("");
  horaPresentacion = ref("");
  clasePromocion = ref(0);
  clasePromovente = ref(0);
  tipoPromovente = 0;
  tipoContenido = ref(0);
  numeroCopias = ref(0);
  numeroAnexo = ref(0);
  secretario = ref(0);
  registroEmpleadoId = 0;
  observaciones = "";
  ipUsuario = "";
  origenPromocion = ref(0);
  numeroRegistro = ref(0);
  numeroOrden = 0;
  conExpedienteElectronico = false;
}

File: Codigo/CJF.Sgcpj.Judicatura/src/Oficialia/Oficialia.Application/Common/Models/InsertarPromocion.cs
namespace CJF.Sgcpj.Judicatura.Application.Common.Models;
public class InsertarPromocion
{
    public long AsuntoNeunId { get; set; }
    public int TipoCuaderno { get; set; }
    public DateTime FechaPresentacion { get; set; }
    public string HoraPresentacion { get; set; }
    public int? ClasePromocion { get; set; }
    public int? ClasePromovente { get; set; }
    public int TipoPromovente { get; set; }
    public int TipoContenido { get; set; }
    public int NumeroCopias { get; set; }
    public int NumeroAnexo { get; set; }
    public int Secretario { get; set; }
    public int RegistroEmpleadoId { get; set; }
    public string Observaciones { get; set; }
    public string IpUsuario { get; set; }
    public int? OrigenPromocion { get; set; }
    public int? NumeroRegistro { get; set; }
    public int? NumeroOrden { get; set; }
    public int? Fojas { get; set; }
    public int? Origen { get; set; }
    public long? Folio { get; set; }
}
